site/docs/components/static-list/accessibility.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Best practices
+
+You'll need to check and test any focusable elements included in the static list content accordingly.
```

Comment:
```suggestion
You must check and test any focusable elements inside the static list content accordingly.
```
?

Reply:
sounds better, will update toast too to avoid replicating this wording in the future.